### ISSUE TITLE
My world Evolution isn't starting - ZipException: Cannot find central directory

### ISSUE THREAD

**Issue by Tobinator1710** [StrangeLoopGames/EcoIssues#21317]
I recently played on a friend's server, later on I want to start my own main project. So when I click start the world nothing happens. But when I start another project of mine, it works.
Pls try to help me,
Tobias

[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/6284698/Player-prev.log)

[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6284702/Player.log)

**Comment by IcedForge** [comment]
Thank you for reporting, im not sure i understand your problem.
Could you fill in some more details regarding your issue?
Do you mean that you cannot start a new server from inside the ingame menu but joining another game works fine?
What operating system are you using? Is the game installed via Steam ? Or from SLG website?



Log Error
```
ZipException: Cannot find central directory
  at ICSharpCode.SharpZipLib.Zip.ZipFile.ReadEntries () [0x00000] in <00000000000000000000000000000000>:0 
  at ICSharpCode.SharpZipLib.Zip.ZipFile..ctor (System.String name) [0x00000] in <00000000000000000000000000000000>:0 
  at LocalServerManager.GetServerLocation (System.String saveName) [0x00000] in <00000000000000000000000000000000>:0 
  at LocalServerManager.LaunchServer (ClientServerStartParameters parameters, System.Action`1[T] onFail, System.String arguments, System.Boolean showGui) [0x00000] in <00000000000000000000000000000000>:0 
  at ServerListingLocalUI.<LoadLocal>b__16_0 () [0x00000] in <00000000000000000000000000000000>:0 
  at Unity.Entities.LambdaSingleJobDescriptionConstructionMethods+WithCodeAction.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
  at PopupDialogUI.OnYesClick () [0x00000] in <00000000000000000000000000000000>:0 
  at Steamworks.ISteamMatchmakingRulesResponse+RulesRefreshComplete.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.Events.InvokableCall.Invoke (System.Object[] args) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.Events.UnityEvent.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
  at System.Data.DataTableNewRowEventHandler.Invoke (System.Object sender, System.Data.DataTableNewRowEventArgs e) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.StandaloneInputModule.ReleaseMouse (UnityEngine.EventSystems.PointerEventData pointerEvent, UnityEngine.GameObject currentOverGo) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.StandaloneInputModule.ProcessMousePress (UnityEngine.EventSystems.PointerInputModule+MouseButtonEventData data) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.StandaloneInputModule.ProcessMouseEvent (System.Int32 id) [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.StandaloneInputModule.Process () [0x00000] in <00000000000000000000000000000000>:0 
  at UnityEngine.EventSystems.EventSystem.Update () [0x00000] in <00000000000000000000000000000000>:0 
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
UnityEngine.EventSystems.StandaloneInputModule:ReleaseMouse(PointerEventData, GameObject)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMousePress(MouseButtonEventData)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent(Int32)
UnityEngine.EventSystems.StandaloneInputModule:Process()
UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by Tobinator1710** [comment]
Okay when I start the game. I click on your worlds, I have a main world and a couple of test worlds. Then I choose my main world (Evolution), press Load and then Yes and the small window "Load this world?" closes but nothing happens.
I've tried many times and every time I get the same result. However, when I tried to start one of my test worlds, they all load immediately.
This problem occurs after playing in a friend's world because the play button no longer exists because my friend's server is not online. So I have to choose one of my worlds. 
I can still create new worlds and join other servers.

My operating System is Windows 10 pro 64 Bit and I installed the game via Steam.

I'm sorry if something is still not clear but I'm not a native speaker and it is a bit challenging for me to express my problem.
If I can give you more information, I'll do my best

Thanks a lot for your help and have a nice day :)

Tobias

**Comment by IcedForge** [comment]
Looks like your save file has broken somehow, this is usually due to not letting it save properly while it shuts down.
Could you try to load a backup of your world? ( eco/eco_data/server/storage/backup ) and replace the Game.eco/Game.db

Are you exiting the game normally via escape menu and disconnect?

**Comment by Tobinator1710** [comment]
I just tried to load a backup of my world and it has been working. So thank you a lot, I'm so happy to be able to play again. :D

Usually I'm exiting the game via escape menu and quit. Is this a problem?

I would also like to note that the longer I play, the frame rate keeps going down. So I got 70+ fps easliy at first and after half and hour it normally dropped to 30-35fps. I'm quiet sure you already knew this but I just want to tell you.

Thank you so much
Tobias
